Misc/NEWS.d/next/Core_and_Builtins/2024-10-11-10-41-05.gh-issue-87790.mlfEGl.rst

```diff
@@ -0,0 +1,3 @@
+Support underscore and comma as thousands separators in the fractional part for
+floating-point presentation types of the new-style string formatting (with
+:func:`format` or :ref:`f-strings`).  Patch by Sergey B Kirpichev.
```

Comment:
Please document also this around: https://docs.python.org/dev/whatsnew/3.14.html#other-language-changes